src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -1,31 +1,36 @@
 import * as React from 'react';
-import { Text } from 'react-native';
+import {Text} from 'react-native';
 
-import { colors } from "../../../utils/colors"
-import { fonts } from "../../../utils/fonts"
+import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
 
 /**
  * @typedef {Object} DiscoveryTitleSeparatorProp
  * @property {String} text
  */
 /**
- * 
- * @param {DiscoveryTitleSeparatorProp} prop 
+ *
+ * @param {DiscoveryTitleSeparatorProp} prop
  */
 const DiscoveryTitleSeparator = (prop) => {
-    const { text } = prop
-    return <Text style={{
+  const {text} = prop;
+  return (
+    <Text
+      style={{
         fontSize: 12,
         lineHeight: 18,
-        fontFamily : fonts.poppins[600],
-        color: colors.black,
+        fontFamily: fonts.poppins[600],
+        color: COLORS.black,
         paddingLeft: 20,
         paddingRight: 20,
         paddingTop: 11,
         paddingBottom: 11,
-        backgroundColor: colors.lightgrey,
+        backgroundColor: COLORS.lightgrey
         // marginTop: 15,
-    }}>{text}</Text>
-}
+      }}>
+      {text}
+    </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The component `DiscoveryTitleSeparator` is using inline styles. This can lead to performance issues as a new style object is created on every render. It's better to use StyleSheet.create outside of the render method.
```diff
-      style={{
-         fontSize: 12,
-         lineHeight: 18,
-        fontFamily: fonts.poppins[600],
-        color: COLORS.black,
-         paddingLeft: 20,
-         paddingRight: 20,
-         paddingTop: 11,
-         paddingBottom: 11,
-        backgroundColor: COLORS.lightgrey
-         // marginTop: 15,
-       }}
+      style={styles.textStyle}
```
And add this after your imports:
```javascript
import {StyleSheet} from 'react-native';

const styles = StyleSheet.create({
  textStyle: {
    fontSize: 12,
    lineHeight: 18,
    fontFamily: fonts.poppins[600],
    color: COLORS.black,
    paddingLeft: 20,
    paddingRight: 20,
    paddingTop: 11,
    paddingBottom: 11,
    backgroundColor: COLORS.lightgrey,
  },
});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->